docs/demos/multi_page_basics/pages/home.py

```diff
@@ -4,10 +4,10 @@
 
 dash.register_page(
     __name__,
-    path="/",
+  #  path="/",
     name="Analytic Apps",
     description="Welcome to my app",
-    order=0,
+  #  order=0,
```

Comment:
Can we take out the commented-out lines, here and in `outlook.py`?

Reply:
OK, done and merged.  Thanks!